src/gui/symbology-ng/qgsgraduatedsymbolrendererv2widget.h

```diff
@@ -86,6 +86,7 @@ class GUI_EXPORT QgsGraduatedSymbolRendererV2Widget : public QgsRendererV2Widget
     void graduatedColumnChanged( QString field );
     void classifyGraduated();
     void reapplyColorRamp();
+    void reapplySizes();
```

Comment:
these changes to qgsgraduatedsymbolrendererv2widget.h should also be propagated to the sip bindings